src/stores/slices/bookmarkSlice.ts

```diff
@@ -0,0 +1,46 @@
+import { createSelector, createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+import { RootState } from "../store";
+
+type Bookmark = {
+  name: string;
+  chainId: string;
+  csvContent: string;
+  transfers: number;
+};
+
+export interface BookmarkState {
+  bookmarksPerChain: Record<string, Bookmark[]>;
```

Comment:
Aren't chainId's usually just `number` type? Why do we choose string here? We could potentially avoid a bunch of `toString()` cast's during creation. Or maybe we are unsure where its coming from and it could be `string | number` and opt for one.

Reply:
on the config service they are returned as string. So I went with string here. But yeah number would also work.